floris/simulation/flow_field.py

```diff
@@ -66,6 +66,22 @@ class FlowField(BaseClass):
         init=False, factory=lambda: np.array([])
     )
 
+    @turbulence_intensities.validator
+    def turbulence_intensities_validator(
+        self, instance: attrs.Attribute, value: NDArrayFloat
+    ) -> None:
+        try:
+            # Check the turbulence intensity is either length 1 or n_findex
+            if len(value) != 1 and len(value) != self.n_findex:
+                raise ValueError("turbulence_intensities should either be length 1 or n_findex")
+        except TypeError as te:
+            # Handle the TypeError here
+            print(f"Caught a TypeError: {te}")
```

Comment:
This should either be printed though the logger or included in the message in the TypeError

Reply:
Done!